cpanfile

```diff
@@ -22,9 +22,15 @@ requires 'XML::Simple'          => '2.00';
 feature examples =>
     ("The example ticketing application" =>
      sub {
+         requires 'DBD::SQLite';
+         requires 'DBI';
```

Comment:
@jonasbn by dropping the web component and switching to the File persister, we can drop the two above and all of the ones in the section below.